src/Robo/Plugin/Commands/CICommands.php

```diff
@@ -91,12 +99,15 @@ public function jobCheckCodingStandards(): Result
         $standards = implode(',', $this->getCodingStandards());
         $extensions = implode(',', $this->phpcsCheckExtensions);
         $ignorePaths = implode(',', $this->phpcsIgnorePaths);
-        return $this->taskExecStack()
-            ->stopOnFail()
-            ->exec('vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer')
-            ->exec("vendor/bin/phpcs --standard=$standards --extensions=$extensions \
-                --ignore=$ignorePaths $this->customCodePaths")
-            ->run();
+        /** @var \Robo\Task\CommandStack $stack */
+        $stack = $this->taskExecStack()->stopOnFail();
+        $stack->exec('vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer');
+        $stack->exec("vendor/bin/phpcs --standard=$standards --extensions=$extensions \
+                --ignore=$ignorePaths $this->customCodePaths");
+        if ($this->lintTwigFiles) {
+            $stack->exec("vendor/bin/twig-cs-fixer lint $this->customCodePaths");
```

Comment:
Since we are adding this, we may want to add an additional alias to the job instead of just `phpcs`. I can't add a suggestion/comment on that line (92), but it would look like this:

```
* @aliases phpcs,twigcs
```